src/systemctl/systemctl.c

```diff
@@ -2877,6 +2877,50 @@ static int start_unit_one(
                         return log_oom();
         }
 
+        r = sd_bus_get_property_trivial(
+                        bus,
+                        "org.freedesktop.systemd1",
+                        unit_path,
+                        "org.freedesktop.systemd1.Unit",
+                        "ConditionResult",
+                        error,
+                        SD_BUS_TYPE_BOOLEAN, &condition_met);
+        if (r < 0)
+                return log_error_errno(r, "Failed to get property ConditionResult: %s", bus_error_message(error, r));
+
+        if (!condition_met) {
+                char *cond, *param;
+                int trigger, negate, state;
+
+                log_warning("The service you attempted to start was skipped:");
```

Comment:
This shouldn't be a warning. A condition not being met is *not* something to warn about or even an error. It's a clean bypass of a service. Hence this should be log_notice() at most.